status-im/status-legacy
# Patch for 8718

[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

Fixes #8718 

### Summary

Defined both translations and languages.
Fixed a JSON error in translations/fr.json

#### Platforms


- Android
- iOS

##### Functional

- 1-1 chats
- public chats
- group chats
- wallet / transactions
- dapps / app browsing
- account recovery
- new account
- user profile updates
- networks
- mailservers
- fleet
- bootnodes

##### Non-functional

- battery performance
- CPU performance / speed of the app
- network consumption

### Steps to test

- Step 1, change device language to something else.
- Step 2, launch Status
- Step 3, verify language changes to the appropriate one.

status: ready
